job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/step/PhaseContext.groovy

```diff
@@ -36,6 +41,13 @@ class PhaseContext extends AbstractContext {
     }
 
     /**
+     * Defines how to run the whole MultiJob phase.
+     */
+    void executionType(String executionType) {
```

Comment:
This feature need version 1.22 of the MultiJob plugin. Add a `@RequiresPlugin` annotation.